test/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "../tsconfig.json",
+	"include": ["."],
+	"compilerOptions": {
+		"noEmit": true
```

Comment:
Why? (Not a complaint, just curious, I am not used to nested `tsconfig.json` files)

Reply:
This doesn't really do anything since we're not running tsc with this config. It's mostly to signal that the files in the `test` directory don't get transpiled. The config is needed because otherwise eslint doesn't know how to apply rules that rely on types - and I changed the root tsconfig to only include `src`.